apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -123,6 +123,7 @@ export function EditableJobPostings({ jobListings }) {
 }
 
 const MIN_CHARS_COUNT = 200
+const MAX_CHARS_COUNT = 255
```

Comment:
I have two comments here:
1. `MIN_CHARS_COUNT` and `MAX_CHARS_COUNT` don't really tell what they are for. Apparently the first one is being used for **summary** and the second one you're adding for **salaryRange**. In that case, we can also mention those field names in the constant names. `SUMMARY_MIN_CHARS_COUNT` and `SALARY_RANTE_MAX_CHARS_COUNT`
2. We can think of a better way of asking salary range, instead of text field with 255 chars limit. But we can do this with Ligia and Asya later.

Reply:
Thanks for the comments,  @helloanil.

1. In some places, we use `MIN_CHARS_COUNT` or `MAX_CHARS_COUNT` variables. In other, we don't have variables at all and pass the numbers directly to the validation rules. Sometimes, we pass the same variable (`MIN_CHARS_COUNT` or `MAX_CHARS_COUNT`) to multiple fields in one component. It would be nice to decide on one consistent approach. What would be the best in all the listed cases?
2. Agree.